scripts/travis_install

```diff
@@ -26,6 +26,7 @@ conda install --yes $DEPS_TRAVIS
 conda config --add channels bokeh
 conda config --add channels javascript   # TODO: move packages to bokeh?
 conda config --add channels mutirri   # TODO: remove when packages are moved to bokeh
+conda config --add channels jrderuiter   # TODO: remove when pytest-cov available on bokeh channel
```

Comment:
Remove that line, the packages are in the bokeh channel now: https://anaconda.org/bokeh/pytest-cov/files